README.Rmd

```diff
@@ -120,16 +137,21 @@ and install using an adapted version of the following commands:
 - For Mac: `install.packages("~/Downloads/iheiddown_macOS.tgz", repos = NULL)`
 - For Unix: `install.packages("~/Downloads/iheiddown_linuxOS.tar.gz", repos = NULL)`
 
-To install from source the latest main version of `{iheiddown}` from Github, 
-please install the `{remotes}` package from CRAN and then enter into the console:
+An easier way of getting access to the development version is to install the
+latest main version of `{iheiddown}` from Github.
+To do that, please install the `{remotes}` package from CRAN and then enter the
+following commands into the console:
 
 - For latest stable version: `remotes::install_github("jhollway/iheiddown")`
 - For latest development version: `remotes::install_github("jhollway/iheiddown@develop")`
```

Comment:
If this is easier (it is, if you have devtools or remotes), then put it above the downloading of the binaries?